staging/src/k8s.io/client-go/tools/clientcmd/api/types.go

```diff
@@ -142,6 +142,15 @@ type AuthInfo struct {
 	// Exec specifies a custom exec-based authentication plugin for the kubernetes cluster.
 	// +optional
 	Exec *ExecConfig `json:"exec,omitempty"`
+
+	// qps indicates the maximum rate tokens will be added to the token bucket for use by a client.
+	// If it's zero, the created RESTClient will use DefaultQPS: 5
+	// Set to -1 to turn off rate limiting.
+	QPS float32 `json:"qps,omitempty"`
+	// burst is the maximum number of tokens to "save up" for immediate use.
+	// If it's zero, the created RESTClient will use DefaultBurst: 10.
+	Burst int32 `json:"burst,omitempty"`
```

Comment:
(though it's not related w/ this pull) the types under this package is supposed to be internal/unversioned, isn't it? but it actually has json tags 🤔